src/vstest.console/Processors/RunSettingsArgumentProcessor.cs

```diff
@@ -197,20 +214,6 @@ private IXPathNavigable GetRunSettingsDocument(string runSettingsFile)
                 runSettingsDocument = MSTestSettingsUtilities.Import(runSettingsFile, runSettingsDocument, Architecture.X86, FrameworkVersion.Framework45);
             }
 
-            if (this.commandLineOptions.EnableCodeCoverage == true)
```

Comment:
Remove this variable from `CommandLineOptions`.

Reply:
We need this variable in `CommandlineOption`.